pallets/creditcoin/src/ocw/errors.rs

```diff
@@ -11,6 +11,7 @@ pub enum OffchainError {
 	InvalidTask(VerificationFailureCause),
 	NoRpcUrl(RpcUrlError),
 	RpcError(RpcError),
+	IncorrectChainId,
```

Comment:
Hmm I guess the risk is just an authority could accidentally have misconfigured its RPCs and therefore be seeing bad data (and producing bad results). An authority could still intentionally use the wrong RPCs, but checking the chain ID at least catches the basic case.

Definitely does feel a bit inefficient, but at the same time the RPC URL can change at any time (since it's pulled from off-chain storage) so just because it was valid at the time of registering a currency doesn't mean it's still valid.

Reply:
Fair. let's keep it until we have voting.